tests/test_gpu_examples.py

```diff
@@ -1168,201 +1168,32 @@ def test_initialize_dora_with_bnb_on_cpu(self, kbit):
         weights_not_cpu = [name for name, p in peft_model.named_parameters() if p.device != torch.device("cpu")]
         assert not weights_not_cpu
 
-    @pytest.mark.single_gpu_tests
-    def test_causal_lm_training_vera(self):
```

Comment:
Hmm, it looks like a bunch of unrelated tests where moved around. I don't think that should be necessary and it also makes reviewing a lot harder. Could you please ensure that the existing tests remain in place and only new tests for GPTQ-QALoRA are added?